bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/RuleRegistryImpl.java

```diff
@@ -32,10 +38,14 @@
     private Set<String> disabledRuledSet = new HashSet(0);
     private Logger logger;
     private Storage<Boolean> disabledRulesStorage;
+    private EventPublisher eventPublisher;
+    
+    private static final String SOURCE = RuleRegistryImpl.class.getSimpleName();
```

Comment:
I don´t think that other components should send rule events. So lets remove the source. A good example for source would be an ItemCommandEvent. The source could be RuleEngine, so you know which commands where executed by the UI and which from a Rule.